test/example/scene_example.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Example do
+  @moduledoc false
+
+  alias Membrane.VideoCompositor.{Canvas, Compound, Scene, Texture}
+  alias Membrane.VideoCompositor.Compound.BaseSize
+  alias Mock.{CornerRounding, Grid, Merging, Overlay, Rotate, ToBall}
+
+  @easy %Scene{
+    alternations: [
+      rounding: %CornerRounding{pixels: 10, degrees: 60}
+    ],
+    layouts: [
+      overlay: Overlay,
+      strip: Grid
+    ],
+    objects: [
+      rounded_1: %Texture{input: :video_1, transformations: [:rounding]},
+      rounded_2: %Texture{input: :video_2, transformations: [:rounding]},
+      rounded_3: %Texture{input: :video_3, transformations: [:rounding]},
+      rounded_4: %Texture{input: :video_4, transformations: [:rounding]},
+      strip: %Compound{
+        base_size: %BaseSize{height: 1080, width: 480},
+        inputs_map: %{
+          rounded_1: :top,
+          rounded_2: :middle,
+          rounded_3: :bottom
+        },
+        layout: :strip
+      },
+      final_object: %Compound{
+        base_size: %BaseSize{height: 1080, width: 1920},
+        inputs_map: %{
+          1.0 => :strip,
+          0.0 => :rounded_4
+        },
```

Comment:
1.0, 0.0? wtf?

Reply:
Using floats as keys in a map is a pretty bad idea, but here it's even worst since it's hard to figure out what that means.

Reply:
1.0 and 0.0 are simply z-values, the strip simply is put on top of rounded 4th video. But I can change that example.